pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/kit/ScanPayload.java

```diff
@@ -8,4 +8,6 @@ public interface ScanPayload {
 
     String getDdpLabel();
 
+    String getHruid();
```

Comment:
**Optional handy trick:**
If new method needs to be added to already existing interface, which may affect to change many files because of it is abstract, you may use `default`, to provide `default` implementation and override it in specific class where you want, other classes which implements this interface will compile and won't ask you to override newly added method.